chainer/functions/loss/softmax_cross_entropy.py

```diff
@@ -111,15 +109,22 @@ def _is_chainerx_supported(self, input_arrays):
         return True
 
     def forward_chainerx(self, inputs):
-        # TODO(niboshi): Current implementation is only intended to support
-        # MNIST example.
         x, t = inputs
```

Comment:
Note that `x.ndim == 2` because of https://github.com/chainer/chainer/pull/8301/files#diff-7f9fda8b243258115ef7bdac145fec8dR106-R107.